ajax/libs/jquery.adaptive-backgrounds/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "author": "Brian Gonzalez",
+  "name": "jquery.adaptive-backgrounds",
+  "description": "A simple jQuery plugin to extract the dominant color of an image and apply it to the background of its parent element.",
+  "version": "1.0.3",
+  "repository": {
+    "type": "git",
+    "url": "http://briangonzalez.github.io/jquery.adaptive-backgrounds.js/"
+  },
+  "filename": "jquery.adaptive-backgrounds.min.js",
+  "homepage": "https://briangonzalez.github.io/jquery.adaptive-backgrounds.js/",
+  "license": "MIT",
+  "keywords": [
+    "jquery",
+    "adaptive",
+    "background"
```

Comment:
Let's add more keywords as the same upstream https://github.com/briangonzalez/jquery.adaptive-backgrounds.js/blob/master/bower.json#L14-L19
How do you like?
Thanks.